reasoner_pydantic/kgraph.py

```diff
@@ -52,6 +53,27 @@ def update(self, other):
             else:
                 self.attributes = other.attributes
 
+class RetrievalSource(BaseModel):
+    """A component of source retrieval provenance"""
+    resource_id: CURIE = Field(
```

Comment:
In the spec this is resource (without _id).

Reply:
The field name is "resource_id". This was updated in TRAPI 1.4 beta 2.

https://github.com/NCATSTranslator/ReasonerAPI/blob/1.4/TranslatorReasonerAPI.yaml

https://github.com/NCATSTranslator/ReasonerAPI/pull/418